docs/website/docs/technologies/tosa.md

```diff
@@ -0,0 +1,15 @@
+# Tensor Operator Set Architecture
+
+[TOSA](https://developer.mlplatform.org/w/tosa/) defines a set of common
+tensor operations to most machine learning
+frameworks. TOSA's design has been focused provide a simple intermediate
+representation for ingesting models from a variety of sources while
+guaranteeing the representation can support generating efficient
+execution on CPUs, GPUs, and custom accelerators. These operations
```

Comment:
long sentence and grammar errors ("has been focused provide")

Reply:
Done.